compiler/AST/foralls.cpp

```diff
@@ -293,6 +293,7 @@ buildFollowLoop(VarSymbol* iter,
                                        zippered,
                                        /*isLoweredForall*/ false,
                                        forallExpr);
+  followBody->orderIndependentSet(true);
```

Comment:
AFAIK no, because this code is called for forall statements and forall expressions; and `forallExpr` indicates it's a forall expression; but then we wouldn't mark it for `forall` statements.